### DISCUSSION TITLE
Week 10 Project Post

### DISCUSSION THREAD

**Discussion by jakegros** [USU-ITLS-x265-23-summer/class-repo#199] [General]
<h1>Week 10 Project Update</h1>
<p>I still have a very long way to go for my site to come together.  It feels a bit daunting with how much still needs to be done. However, i just try to keep chipping away and keep the momentum going.</p>

<h2>Progress made:</h2>
<ul>
<li>[x] Added more pages to my site 
<li>[x] Started to add buttons
<li>[x] Buttons link to new pages
<li>[x] Added CSS selectors for heading 
<li>[x] Inline logo and title 
<li>[x] Started contact form </li>
</ul>

<h2>Next steps:</h2>
<li>Build up homepage 
<li>More CSS styling 
<li>Add the other pages
<li>Finish adding content
<li>Finish form
<li>Add logo to each page and link it to home 
<li>Semantic html 
<li>Dropdown?
<li>Start incorporating JavaScript </li>
</ul>

**Comment by CJ3315** [comment]
What are you thinking of doing for your contact form? I'm having trouble starting mine.

**Reply by jakegros** [reply]
Initially, I was thinking of something very basic where someone could fill out the form to learn more. Essentially, a client making a request. 
I am now wondering if there is a way to incorporate JavaScript to make it better.

**Comment by CJ3315** [comment]
Okay yeah I was thinking the same thing, I'll let you know if I did anything with Java

**Comment by Skye404** [comment]
I added a fillable form to my project. I'm hoping it can work as a place for people to send an email directly to me. Let me know if you find any helpful tips/tricks with the forms. Mine looks pretty basic right now.

**Reply by jakegros** [reply]
Right on. Another thing I realized is JavaScript can be used to hide elements
